src/java.desktop/share/classes/com/sun/imageio/plugins/bmp/BMPImageReader.java

```diff
@@ -2039,17 +2037,9 @@ public void readAborted(ImageReader src) {}
 
     private static Boolean isLinkedProfileAllowed = null;
 
-    @SuppressWarnings("removal")
     private static boolean isLinkedProfileAllowed() {
         if (isLinkedProfileAllowed == null) {
-            PrivilegedAction<Boolean> a = new PrivilegedAction<Boolean>() {
-                @Override
-                public Boolean run() {
-                    return Boolean.
-                        getBoolean("sun.imageio.bmp.enableLinkedProfiles");
-                }
-            };
-            isLinkedProfileAllowed = AccessController.doPrivileged(a);
+            isLinkedProfileAllowed = Boolean.getBoolean("sun.imageio.bmp.enableLinkedProfiles");
```

Comment:
Do we even need that static variable? Why not just
`return Boolean.getBoolean("sun.imageio.bmp.enableLinkedProfiles");`
Performance? Does it deserve an internal caching? Just curious.